services/elections.request.service.ts

```diff
@@ -0,0 +1,340 @@
+  updatePosition(data: any, queryParams: any): Observable<Position> {
+    const positionObservable = super.put(`${this.baseURL}/position/` + queryParams, data);
+    return positionObservable;
+  }
+
+  ///////////////////
+  // Candidates
+  //////////////////
+  /**
+   * List candidates
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/candidate/get_election__election_id__candidate
+   * @param queryParams
+   * @return an observable of all elections or elections specified by urlParams
+   */
+  listCandidates( queryParams: any, jsonData?: any): Observable<Candidate[]> {
+    const candidateObservable = super.get(`${this.baseURL}/election/${queryParams}/candidate`, jsonData).pipe(
+      map((data: {candidates: Candidate[]}) => data.candidates)
+    );
+    return candidateObservable;
+  }
+
+  /**
+   * Read candidates
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/candidate/get_election__election_id__candidate__candidate_id_
+   * @param queryParams
+   * @param candidateId
+   * @return a single candidate JSON object
+   */
+  readCandidate(queryParams: any, candidateId: any): Observable<Candidate> {
+    const candidateObservable = super.get(`${this.baseURL}/election/${queryParams}/candidate/${candidateId}`).pipe(
+      map((data: {candidates: Candidate}) => data.candidates)
+    );
+    return candidateObservable;
+  }
+
+  /**
+   * Create candidate
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/candidate/post_election__election_id__candidate
+   * @param queryParams
+   * @return JSON object containing the candidate info created
+   */
+  createCandidate( queryParams: any, data: any): Observable<Candidate> {
+    const candidatesObservable = super.post(`${this.baseURL}/election/${queryParams}/candidate`, data);
+    return candidatesObservable;
+  }
+
+  /**
+   * Update Candidate
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/candidate/put_election__election_id__candidate__candidate_id_
+   * @param data
+   * @param queryParams
+   * @param candidateID
+   * @return JSON object containing the candidate info that was updated
+   */
+  updateCandidate(data: any, queryParams: any, candidateID: any): Observable<Candidate> {
+    const candidateObservable = super.put(`${this.baseURL}/election/${queryParams}/candidate/${candidateID}`, data);
+    return candidateObservable;
+  }
+
+  /**
+   * Remove candidate
+   *
+   * https://docs.aswwu.com/?url=https://raw.githubusercontent.com/ASWWU-Web/python_server/develop/docs/elections.yml#/candidate/delete_election__election_id__candidate__candidate_id_
+   * @param queryParams
+   * @param candidateID
+   */
+  removeCandidate(queryParams: any, candidateID: any): Observable<[]> {
```

Comment:
This is probably a good way to return an empty observable, but it might be good to mess with it. For instance we might want to return a type `Observable<>` instead. Or It might be better to see what it returns ignoring the type annotation and then just set the type annotation to that.

Reply:
When attempting to test Observable<>, there was an error stating that there needed to be at least 1 argument.